components/HomeScreen/HomeScreen.jsx

```diff
@@ -0,0 +1,11 @@
+import { Text, SafeAreaView } from "react-native";
+
+const HomeScreen = () => {
+  return(
+    <SafeAreaView>
```

Comment:
Instead of using the SafeAreaView we suggest using the "react-native-safe-area-context" library.[ This libraries explanation goes into a bit of detail about a few of the reasons, but you'll generally have a much better and easier to control experience. ](https://reactnavigation.org/docs/handling-safe-area/)